roles/repository/tasks/main.yml

```
@@ -128,18 +128,19 @@
 
 - name: Download AMPS
   become: true
-  loop: "{{ amp_downloads }}"
+  loop: "{{ repository_amp_downloads + repository_extra_amp_downloads }}"
   ansible.builtin.get_url:
     url: "{{ item.url }}"
-    checksum: sha1:{{ lookup('url', item.sha1_checksum_url, username=nexus_user, password=nexus_password) }}
+    checksum: "{{ item.checksum }}"
     dest: "{{ item.dest }}"
     owner: "{{ username }}"
     group: "{{ group_name }}"
     mode: 'u=rw,g=r,o=r'
-    url_username: "{{ nexus_user }}"
-    url_password: "{{ nexus_password }}"
+    url_username: "{{ item.url_username }}"
+    url_password: "{{ item.url_password }}"
   register: amp_download_result
   async: 900
+  when: item.enabled | default(True) # when adding additional amp no need to specify enabled
```

```suggestion
  when: item.enabled | default(true) # when adding additional amp no need to specify enabled
```

strange that ansible lint is not complaining because I remember it used to